test/test_sumArrayExample.cpp

```diff
@@ -0,0 +1,225 @@
+#include "yacx/main.hpp"
+#include <catch2/catch.hpp>
+
+using yacx::Kernel, yacx::Source, yacx::KernelArg, yacx::Options, yacx::Device, yacx::type_of;
+
+
+TEST_CASE("sumArray", "[example_program]") {
+
+  SECTION("SUM_ARRAY WITH 1 and fixed values") {
+
+
+        Device device;
+        Options options{yacx::options::GpuArchitecture(device),
+                        yacx::options::FMAD(false)};
+        options.insert("--std", "c++14");
+        Source source{"extern \"C\" __global__ void sumArrayOnGPU(float *A, float *B, float* C){\n"
+                        "int i_inBlock=threadIdx.x+threadIdx.y*blockDim.x+threadIdx.z*blockDim.y*blockDim.x;\n"
+                        "int blockID= blockIdx.x;\n"
+                        "int i = i_inBlock + blockID*(blockDim.x*blockDim.y*blockDim.z);\n"
+                        "C[i]=A[i]+B[i];\n"
+                        "}\n"
+        };
```

Comment:
das könnte eig auch außerhalb der Section, da es jeder Section aufgerufen werden soll

Reply:
Da stimme ich absolut zu. Danke :) Ich habe es in einem neuen Commit angepasst